src/definitions/general_config.rs

```diff
@@ -2,7 +2,11 @@ use crate::{business_logic::state::state_api_objects::BlockInfo, utils::Address}
 use felt::Felt;
 use getset::{CopyGetters, MutGetters};
 use num_traits::Zero;
+use std::collections::HashMap;
 
+use super::error::StarknetChainIdError;
+
+#[allow(unused)]
```

Comment:
I don't think you need this here, but if it's really needed it's better to use `#[allow(dead_code)]`.

Reply:
i delete the module as it is unused

Reply:
I think I was using it.

Reply:
i used it for something before, but it was not necessary after all. are you sure that you have used it?

Reply:
Yes, in `tests/syscalls.rs`. I guess I could've used `starknet_api`'s version, but I assumed we'd like to remove that dependency once we're ready.

Reply:
i checked it in this branch, but what i deleted was the `error.rs` module the one you used is still there.

Reply:
Oh ok, although this comment was referring to the `#[allow(unused)]`, not the `use super::error::StarknetChainIdError;` line. That line only appeared there because github selects minimum 3 lines of code, and therefore I couldn't select only the unused attribute.